content/flux/v0/stdlib/universe/cumulativesum.md

```diff
@@ -10,6 +10,8 @@ menu:
 weight: 101
 flux/v0/tags: [transformations]
 introduced: 0.7.0
+aliases:
+  - /flux/v0/stdlib/built-in/transformations/cumulativesum/
```

Comment:
Since the Flux stdlib docs are generated, we have a deploy script that injects additional "non-standard" frontmatter  into each stdlib docs. Rather than update this here, updated it `data/flux_stdlib_frontmatter.yml`. The additional frontmatter for this page starts on [line 2664](https://github.com/influxdata/docs-v2/blob/master/data/flux_stdlib_frontmatter.yml#L2664-L2672).